qml/KeyboardContainer.qml

```diff
@@ -58,6 +59,14 @@ Item {
             }
         }
     }
+    Loader {
+        id: symbolKeypadLoader
+        objectName: "characterKeyPadLoader"
+        anchors.fill: parent
+        asynchronous: true
+        visible: panel.state !== "CHARACTERS"
```

Comment:
what do you think of changing this to `panel.state === "SYMBOLS"` so that more layouts could be defined like is done in #129